### ISSUE TITLE
make search available for other movies on TMDb

### ISSUE THREAD

**Issue by theDavidBarton** [theDavidBarton/trending-movies-react-app#32]
make search available for other movies on TMDb

**Comment by theDavidBarton** [comment]
api part implemented with #33 usage:
`GET /api/movieAutocomplete?q=${query}`

**Comment by theDavidBarton** [comment]
mocked response for `q=shining` :
```
{"page":1,"total_results":65,"total_pages":4,"results":[{"popularity":30.643,"vote_count":9641,"video":false,"poster_path":"/9fgh3Ns1iRzlQNYuJyK0ARQZU7w.jpg","id":694,"adult":false,"backdrop_path":"/h4DcDCOkQBENWBJZjNlPv3adQfM.jpg","original_language":"en","original_title":"The Shining","genre_ids":[27,53],"title":"The Shining","vote_average":8.2,"overview":"Jack Torrance accepts a caretaker job at the Overlook Hotel, where he, along with his wife Wendy and their son Danny, must live isolated from the rest of the world for the winter. But they aren't prepared for the madness that lurks within.","release_date":"1980-05-22"},{"popularity":5.769,"vote_count":74,"video":false,"poster_path":"/e3D4a0B7BbzJfDMuLurDvxZP2oY.jpg","id":31962,"adult":false,"backdrop_path":"/tUevbHi0u6t1VailNVOj0b9LTuW.jpg","original_language":"en","original_title":"Shining Through","genre_ids":[18,10749],"title":"Shining Through","vote_average":6.4,"overview":"Spirited New Yorker Linda Voss goes to work for international lawyer and secret Office of Strategic Services operative Ed Leland just before World War II. As they fall in love, the United States enters the fight against Hitler, and Linda volunteers to work for Ed spying undercover behind Nazi lines. Assigned to uncover information about a German bomb, Linda also has personal motives to fulfill: discovering the fate of her Jewish family members in Berlin.","release_date":"1992-01-31"},{"popularity":5.087,"id":18875,"video":false,"vote_count":53,"vote_average":7.1,"title":"Making 'The Shining'","release_date":"1980-10-04","original_language":"en","original_title":"Making 'The Shining'","genre_ids":[99],"backdrop_path":"/6fwbN3OeoiN1Dpx1ZBfnei78hJ4.jpg","adult":false,"overview":"Directed and Edited by Vivian Kubrick, Stanley Kubrick's daughter, this film offers a look behind the scenes during the making of The Shining.","poster_path":"/s1Uc8J4wpXTDKTJ08dKzlHVTc0q.jpg"},{"popularity":1.123,"id":111463,"video":false,"vote_count":4,"vote_average":4.5,"title":"The Shining Hour","release_date":"1938-11-18","original_language":"en","original_title":"The Shining Hour","genre_ids":[18,10749],"backdrop_path":"/6mqZynDIk2I3wRLowLuKBpMbDfO.jpg","adult":false,"overview":"A nightclub dancer marries into a rich family, and some of its members resent her.","poster_path":"/icPlJYyRg3zhvONL1wwbqbbXXpH.jpg"},{"popularity":1.953,"id":13923,"video":false,"vote_count":11,"vote_average":5.9,"title":"A Bright Shining Lie","release_date":"1998-01-01","original_language":"en","original_title":"A Bright Shining Lie","genre_ids":[18,10752],"backdrop_path":"/foXUmyb15Iq4K7AS5xGH4E6gres.jpg","adult":false,"overview":"Something in his past keeps career Army man John Paul Vann from advancing past colonel. He views being sent to Vietnam as part of the US military advisory force a stepping stone to promotion. However, he disagrees vocally (and on the record) with the way the war is being run and is forced to leave the military. Returning to Vietnam as a civilian working with the Army, he comes to despise some South Vietnamese officers while he takes charge of some of the U.S. forces and continues his liaisons with Vietnamese women.","poster_path":"/1cbOs8MZGn02alFrzv6DXVDValL.jpg"},{"popularity":1.515,"id":267574,"video":false,"vote_count":0,"vote_average":0,"title":"The Shining Forwards and Backwards","release_date":"2011-03-09","original_language":"en","original_title":"The Shining Forwards and Backwards","genre_ids":[27],"backdrop_path":null,"adult":false,"overview":"An experimental film comprised of Stanley Kubrick's THE SHINING played forwards and backwards at the same time on the same screen, creating bizarre juxtapositions and startling synchronicities","poster_path":"/xCKC05tQWItZLx1YuchxwOlGVaV.jpg"},{"popularity":1.031,"id":182225,"video":false,"vote_count":0,"vote_average":0,"title":"The Shining Code 2.0","release_date":"2012-01-01","original_language":"en","original_title":"The Shining Code 2.0","genre_ids":[],"backdrop_path":null,"adult":false,"overview":"The makers of \"The Shining Code 2.0\" are one of five featured conspiracists in \"Room 237\" who’ve all ascribed different meanings and markers and signifiers to Stanley Kubrick’s 1980 adaptation of Stephen King’s \"The Shining\", in their case that the filmmaker’s ostensible horror movie is actually a coded confession regarding his participation in the faking of the Apollo moon landings...","poster_path":"/qrGahGGIQCzB7twh47qoxdVlbh7.jpg"},{"popularity":0.629,"id":475008,"video":false,"vote_count":0,"vote_average":0,"title":"Shining the Ball","release_date":"2005-06-10","original_language":"en","original_title":"Shining the Ball","genre_ids":[18],"backdrop_path":null,"adult":false,"overview":"An attractive young man joins a cricket club and finds that the vice captain may have an unexpected vice...","poster_path":null},{"popularity":0.6,"id":103278,"video":true,"vote_count":3,"vote_average":6.2,"title":"Shining: Live Blackjazz","release_date":"2011-11-11","original_language":"no","original_title":"Shining: Live Blackjazz","genre_ids":[10402],"backdrop_path":"/rx4idr6DdMVEQjzLY4CiqtDAdCl.jpg","adult":false,"overview":"Fronted and produced by Jorgen Munkeby, mixed by Sean Beavan (Nine Inch Nails, Slayer, Marilyn Manson) and mastered by Tom Baker (Nine Inch Nails, Marilyn manson), “Live Blackjazz” pushes the boundaries of both metal and jazz genres to facilitate a whole new experience of listening to both. The beauty of music such as this is that it will test the patience of the metal fan and the fan of experimental jazz. There are sections of saxophone that are reminiscent of early free jazz, and then there are sections of keyboard which are reminiscent of early 1970’s progressive rock, followed by grandeur sections of choral vocal which, just when you thought you had the measure of the piece, launch into frantic, grindcore metal passages.","poster_path":"/jaKD9bpGCplMShrX6ALWpqrqDlC.jpg"},{"popularity":1.619,"id":228633,"video":false,"vote_count":3,"vote_average":7.7,"title":"View from the Overlook: Crafting 'The Shining'","release_date":"2007-10-23","original_language":"en","original_title":"View from the Overlook: Crafting 'The Shining'","genre_ids":[99],"backdrop_path":"/tPPiZNV7UFWQZHBcE8EMmf62xmO.jpg","adult":false,"overview":"An overview of the making of, and a look at the continuing popularity of, Stanley Kubrick's The Shining.","poster_path":"/mAZhXkepx0UpvQxMWaPVT6AHCOO.jpg"},{"popularity":0.635,"vote_count":1,"video":false,"poster_path":"/msbBF43KAcLZi3VxwlKhTEpSoii.jpg","id":446584,"adult":false,"backdrop_path":null,"original_language":"en","original_title":"A Shining Season","genre_ids":[18],"title":"A Shining Season","vote_average":4,"overview":"The true story of University of New Mexico track star John Baker who, dying of cancer, spent his remaining time coaching a losing girls\" track team to a winning season.","release_date":"1979-12-26"},{"popularity":0.6,"id":570549,"video":false,"vote_count":0,"vote_average":0,"title":"Blue Shining","release_date":"2015-06-05","original_language":"en","original_title":"Blue Shining","genre_ids":[],"backdrop_path":null,"adult":false,"overview":"What would The Shining look like had it been directed by David Lynch? Would it be a dream or a nightmare? Blue Shining combines both worlds in a playful manner by integrating elements from Lynch's films into Kubrick's movie to give the Stephen King classic a Lynchian atmosphere. Can you find all the hidden items, including the blue key from Mulholland Dr.?... Enjoy!","poster_path":null},{"popularity":0.6,"id":261356,"video":false,"vote_count":0,"vote_average":0,"title":"Still Shining","release_date":"","original_language":"en","original_title":"Still Shining","genre_ids":[],"backdrop_path":null,"adult":false,"overview":"Created in 2006, this remembrance piece is created as a tribute to the memory and legacy of James \"J.Dilla\" Yancey. This is a piece designed for his fans and supporters who knew of his accomplishments before February 2006 and those that have grown to appreciate his genius. Here, we gain a greater insight and understanding about our musical icon.","poster_path":null},{"popularity":0.733,"id":256371,"video":false,"vote_count":4,"vote_average":7.6,"title":"Shining Victory","release_date":"1941-05-30","original_language":"en","original_title":"Shining Victory","genre_ids":[18],"backdrop_path":"/nQhRhbwklqATi6iUaI1CMu2OF0T.jpg","adult":false,"overview":"A psychiatrist sacrifices everything for his research.","poster_path":"/93p074mZN47yPJiS3ktFjsdT4sH.jpg"},{"popularity":0.6,"id":334034,"video":true,"vote_count":1,"vote_average":10,"title":"Shining: Live at Øya Festival","release_date":"2013-07-24","original_language":"no","original_title":"Shining: Live at Øya Festival","genre_ids":[10402],"backdrop_path":"/pnJJjKZH00UULmnqUjfanJnNK6R.jpg","adult":false,"overview":"This is the entire set from Norway's Øya Festival in Oslo.","poster_path":"/7J3NIiCtCcjhxvFohPK0RAZTMUt.jpg"},{"popularity":0.638,"id":337285,"video":false,"vote_count":0,"vote_average":0,"title":"Earth Wind & Fire: Shining Stars","release_date":"2001-08-21","original_language":"en","original_title":"Earth Wind & Fire: Shining Stars","genre_ids":[],"backdrop_path":null,"adult":false,"overview":"This is their story. This is their music...There is nowhere else you can go to see and hear the amazing stories and songs that form the history of Earth Wind &amp; Fire. The Earth Wind &amp; Fire story is told by the people who created it, lived it and made it happen.","poster_path":"/8ABs9LYRqVp3sDhyf1kcL9VRDBB.jpg"},{"popularity":0.6,"id":257402,"video":false,"vote_count":0,"vote_average":0,"title":"The Shining Future","release_date":"1944-04-11","original_language":"en","original_title":"The Shining Future","genre_ids":[10402,99],"backdrop_path":null,"adult":false,"overview":"Documentary short film intended to drum up support for the Fifth War Loan Campaign. It shows a happy family in the future of 1960 enjoying the prosperity and advantages made possible by the successful prosecution of the war, and how the sacrifices of 1944 have made the world a better place.","poster_path":"/4NA3Gz1ALgHwHGlCTtUkneTzNbh.jpg"},{"popularity":0.6,"id":501054,"video":false,"vote_count":0,"vote_average":0,"title":"Cold Body Shining","release_date":"2017-01-26","original_language":"pl","original_title":"Cold Body Shining","genre_ids":[],"backdrop_path":null,"adult":false,"overview":"","poster_path":null},{"popularity":0.689,"id":392507,"video":false,"vote_count":1,"vote_average":5,"title":"The Shining Star of Losers Everywhere","release_date":"2016-05-20","original_language":"en","original_title":"The Shining Star of Losers Everywhere","genre_ids":[99],"backdrop_path":null,"adult":false,"overview":"In Japanese, Haru Urara means “Glorious Spring.” But as this 30 for 30 Short, directed by Mickey Duzyj, makes delightfully clear, the name means a great deal more to the Japanese people. Haru Urara was a thoroughbred mare who never won a race. She first came to the public’s attention after her 88th straight loss at troubled Kochi Racetrack, then picked up more and more supporters with each defeat. With her pink Hello Kitty patches, she became a symbol of hope in Japan. And though she never did win, Haru Urara did accomplish something greater: she saved the racetrack.","poster_path":"/wfAvsmcEjKXkYxPEb1zb3KEuuaL.jpg"},{"popularity":1.146,"id":173458,"video":false,"vote_count":4,"vote_average":6.1,"title":"Stephen King: Shining in the Dark","release_date":"1999-09-09","original_language":"en","original_title":"Stephen King: Shining in the Dark","genre_ids":[99],"backdrop_path":null,"adult":false,"overview":"A documentary looking at the life and work of Stephen King.","poster_path":"/1F0bQ77FBIEk0PbS3vV8mbLgZSj.jpg"}]}
```
